deploy/lvm-operator.yaml

```diff
@@ -1080,6 +1080,23 @@ roleRef:
   name: openebs-lvm-provisioner-role
   apiGroup: rbac.authorization.k8s.io
 
+---
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: openebs-lvm-localpv-csi-controller-critical
+value: 900000000
+globalDefault: false
+description: "This priority class should be used for the OpenEBS LVM localPV CSI driver controller deployment only."
+---
+apiVersion: scheduling.k8s.io/v1
```

Comment:
move this to node agent section below.

Reply:
Priorityclass should be exists before creating the sts using it.